### ISSUE TITLE
How to Successfully move a site from IIS to Linux

### ISSUE THREAD

**Issue by JExactly** [snipe/snipe-it#2823]
#### Expected Behavior (or desired behavior if a feature request)

Functional site after moving it and database to Linux server

-----

#### Actual Behavior

I am getting a "Whoops, looks like something went wrong." page when opening the site.

What I Have done that appears to be successful.

1.) Cloned Snipe-IT from Git.
2.) walked through all of the standard install steps for linux.
3.)applied the exact key that was generated from my .env file in IIS to .env in linux.
4. Applied permissions for my www user to the storage and public/uploads folders
5.) Ran all the usual "Upgrade" commands since the site DB was imported from V3.3 running site and the current version is 3.5/3.5.1

Getting above error.  Does anyone have any good ideas on what I should have done, should be doing?

-----

#### Please confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode
- [ ] I have read [checked the Common Issues page](http://docs.snipeitapp.com/common-issues.html)

-----
#### Please provide answers to these questions before posting your bug report:

- Version of Snipe-IT you're running
- What OS and web server you're running Snipe-IT on
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
- If you're getting an error in your browser, include that error
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is [reproduceable on the demo](https://snipeitapp.com/demo).
- Include any additional information you can find in `app/storage/logs` and your webserver's logs.
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

**Comment by snipe** [comment]
Please enable debug mode so we can see the actual errors.

**Comment by JExactly** [comment]
Once I enabled the Debug (Didn't think of it when I posted) Wroking on 8 things at the same time per usual... I was able to resolve once I stepped away from it for a few hours.. Had a few issues. 
1.) LDAP settings issues from when it was running in IIS and on a domain Server, ( manually modified my MySQL table value back to 0 so I could login with my local account) Realized after I sorted this that you created an artisan command.
2.) had a permissions issue for a few of the folders I thought I already sorted. 
3.) Also did not have the php-ldap module installed. 
I appreciate you pointing me in the right direction. Otherwise it seems to be up and running and I have all issues sorted at the moment. If I run into anymore and get stumped, I know where to go. Thanks again for your help @snipe !! Kindly appreciated. Otherwise, I would like to state, it runs much smoother and faster on my linux box than it does on my IIS Server. (both running in Hyper-V btw). We will see how it performs once there is a heavier load on apache with users pegging my other hosted app on the same web server.
